src/test/Testcafe/Instantiation_tests/instantiation_tests.js

```diff
@@ -8,12 +8,17 @@ let app = new App();
 let designer = new DesignerPage();
 let runtime = new RunTimePage();
 
-fixture('Instantiation tests')
+/*
```

Comment:
should the file name be same as other test .js files ? instantiation-tests.js ?

Reply:
fixed